### ISSUE TITLE
The latest packaging of Linux platform is built on ardent

### ISSUE THREAD

**Issue by minggangw** [ros2/ci#123]
Hi,

I noticed that the latest successful Linux build, [build 970](http://ci.ros2.org/view/packaging/job/packaging_linux/970/), is against ardent. Have the ci policy changed?

I want to download the latest packaging which is on master branch to verify our project `rclnodejs` and `ros2-web-bridge`, but we met the [problem](https://travis-ci.org/RobotWebTools/ros2-web-bridge/builds/340035246?utm_source=github_status&utm_medium=notification) that the code of `rcl` was not on the master branch. Where can I get the master packaging?

Thanks!

**Comment by mikaelarguedas** [comment]
TL;DR the ci policy didnt change and we will keep providing nightly builds from master. You got unlucky and got the packaging job used for the Ardent patch release before the nightly job kicked in.


Yeah I think you just got unlucky: we use the packaging jobs to packages our official releases as well as to produce nightlies archive for users to use. We just released the first patch release of Ardent. So I think that it was the "last successful build" when you looked it up.
As the packaging job runs every 24hours you must have fallen in the time period between the manually triggered ardent build and the nightly master build. When trying it now the [last successful build](http://ci.ros2.org/view/packaging/job/packaging_linux/lastSuccessfulBuild/) points to 972 that is from the 
 master branch.

Thanks for reporting!

I'm going to close this as this is already "fixed" but we will keep it in mind next time we cut a release to avoid this inconvenience. Feel free to comment on this closed ticket if you still face a problem and we can reopen it

**Comment by minggangw** [comment]
Thanks for the detailed explanation.
